stablehlo/reference/Ops.cpp

```diff
@@ -1289,17 +1324,20 @@ SmallVector<Tensor> evalScatterOp(
     auto resultIndex = fullStartIndex + fullWindowIndex;
     if (!resultIndex.inBounds(results[0].getShape())) continue;
 
-    SmallVector<Tensor> updateComputationArgs;
-    for (auto result : results)
-      updateComputationArgs.push_back(
-          Tensor(RankedTensorType::get({}, result.getElementType()),
-                 result.get(resultIndex)));
-    for (auto update : updates)
-      updateComputationArgs.push_back(
-          Tensor(RankedTensorType::get({}, update.getElementType()),
-                 update.get(updateIndex)));
-
-    auto updatedValues = eval(updateComputation, updateComputationArgs, &scope);
+    SmallVector<StablehloValue> updateComputationArgs;
+    for (auto result : results) {
+      Tensor tensor(RankedTensorType::get({}, result.getElementType()),
+                    result.get(resultIndex));
+      updateComputationArgs.push_back(StablehloValue(tensor));
```

Comment:
There's an easier way to adapt to these changes - instead of push_back, we can emplace_back. This way, we'll keep the existing code and won't need to introduce temporary variables that we'll need to somehow name, braces that we'll need to spell out, etc. (Same below).

Reply:
Good call, I've also updated other occurrences of push_back with unnecessary copies for Tensors into emplace_back.